bin/ch/Debugger.cpp

```diff
@@ -175,10 +175,9 @@ JsValueRef Debugger::Evaluate(JsValueRef callee, bool isConstructCall, JsValueRe
     {
         IfJsErrorFailLogAndRet(ChakraRTInterface::JsNumberToInt(arguments[1], &stackFrameIndex));
 
-        AutoString argstr;
-        size_t length;
-        IfJsErrorFailLogAndRet(ChakraRTInterface::JsValueToCharCopy(arguments[2], &argstr, &length));
-        ChakraRTInterface::JsDiagEvaluateUtf8(*argstr, stackFrameIndex, &result);
+        AutoString argstr(arguments[2]);
+        IfJsErrorFailLogAndRet(argstr.GetError());
+        ChakraRTInterface::JsDiagEvaluateUtf8(argstr.GetString(), stackFrameIndex, &result);
```

Comment:
@agarwal-sandeep This method; we use on CH temporarily and one of the methods we have left from the experimental phase. While we are enabling Debug on xplat, we should take care of this part too though.